applications/app/views/fragments/audioBody.scala.html

```diff
@@ -0,0 +1,179 @@
+
```

Comment:
You can drop that file entirely. In `mediaBody.scala.html`, you just need to change three things:
1. `import experiments.{ActiveExperiments, AudioPageChange }`
1. line [14](https://github.com/guardian/frontend/blob/master/applications/app/views/fragments/mediaBody.scala.html#L14), add an item to the `Map`:
```scala
"new-audio" -> ActiveExperiments.isParticipating(AudioPageChange)
```
1. line [35](https://github.com/guardian/frontend/blob/master/applications/app/views/fragments/mediaBody.scala.html#L35), add a case:
```scala
case audio: Audio if (ActiveExperiments.isParticipating(AudioPageChange)) => {
  <figure data-component="main audio new-player"
    id="audio-component-container"
    data-media-id="@audio.elements.mainAudio.map{ audioElement => @audioElement.properties.id}"
    data-source="@audio.downloadUrl.map(Audio.acastUrl)"
    data-itunes-url="@audio.iTunesSubscriptionUrl.getOrElse("#")"
    data-download-url="@audio.downloadUrl.getOrElse("#")">
</figure>
```

Reply:
That will allow you to undo the changes in `MediaController` and `ContentHtmlPage`

Reply:
Fair point. However, this choice comes from earlier work me and Tom did to split out the Audio Page from the Video Page, anticipating the full redesign of the Audio page. 
In order to make life easier for the full redesign of the Audio page I'm inclined to keep the splitting in the `ContentHtmlPage` so we can completely delete the `audioBody.scala.html` and start again from scratch.